CAM-14558: chore(release): edit 7.18 update guide (#1313)
camunda/camunda-docs-manual: content/update/minor/717-to-718/was.md

If you have previously replaced the default Camunda Platform configuration with a custom configuration following any of 
the methods outlined in the [deployment descriptor reference][configuration-location], it may be necessary to restore 
this configuration. This can be done by repeating the configuration replacement steps for the updated platform.

# 5. Install the Camunda Archive

Install the Camunda EAR, or the file `$WAS_DISTRIBUTION/modules/camunda-ibm-websphere-ear-{{< minor-version >}}.0-ee.ear`.

* During the installation on WebSphere 9, the EAR will try to reference the `Camunda` shared library.
* On WebSphere Liberty, please follow [the EAR installation guide]({{< ref "installation/full/was/manual-liberty.md#camunda-platform-ear" >}}) 
to deploy the Camunda EAR correctly.

# 6. Install the web applications

## REST API

The following steps are required to update the Camunda REST API on an IBM WebSphere instance:

* On WebSphere 9:
  1. Deploy the web application `$WAS_DISTRIBUTION/webapps/camunda-engine-rest-{{< minor-version >}}.0-ee-was.war` to your IBM WebSphere instance.
  2. Associate the web application with the `Camunda` shared library.
* On WebSphere Liberty:

>   1. Place the web application `$WAS_DISTRIBUTION/webapps/camunda-engine-rest-{{< minor-version >}}.0-ee-was.war` on you Liberty `$YOUR_SERVER/apps/` directory/.

Reviewer: The slash at the end is probably not needed and there is a typo with 'on you'.

```suggestion
  1. Place the web application `$WAS_DISTRIBUTION/webapps/camunda-engine-rest-{{< minor-version >}}.0-ee-was.war` in the Liberty `$YOUR_SERVER/apps/` directory.
```

  2. Configure the `server.xml` as described in [the Liberty installation guide]({{< ref "installation/full/was/manual-liberty.md#rest-api" >}}).

## Cockpit, Tasklist, and Admin

The following steps are required to update the Camunda web applications Cockpit, Tasklist, and Admin on an IBM WebSphere instance:

* On WebSphere 9:
  1. Deploy the web application `$WAS_DISTRIBUTION/webapps/camunda-webapp-ee-was-{{< minor-version >}}.0-ee.war` to your IBM WebSphere instance.
  2. Associate the web application with the `Camunda` shared library.
* On WebSphere Liberty:

>   1. Place the web application `$WAS_DISTRIBUTION/webapps/camunda-webapp-ee-was-{{< minor-version >}}.0-ee.war` on you Liberty `$YOUR_SERVER/apps/` directory/.

Reviewer: ```suggestion
  1. Place the web application `$WAS_DISTRIBUTION/webapps/camunda-webapp-ee-was-{{< minor-version >}}.0-ee.war` in the Liberty `$YOUR_SERVER/apps/` directory.
```
